### DISCUSSION TITLE
1.1.1 plotting going well

### DISCUSSION THREAD

**Discussion by Teledyol** [Chia-Network/chia-blockchain#2628] [General]
The previous versions were so buggy I reinstalled 1.0.0 to clean up the mess. The .4 and .5 installations were leaving persistent artifacts and files on the temp drive. I was reticent about installing 1.1, However…

I ran the "Queue Hack" for 4x5 in parallel and things ran like a charm! The most amazing thing was the lack of other "errors" for that whole duration; very few, in comparison. That's a big improvement. So, I've cleaned up the mess a bit and am trying  4 x 10 to see if I can actually ignore the plotting for a few days.

Chia is still running after the blockchain GUI app is shut down, which should not be happening. But I've seen that reported elsewhere

Here's the rig: https://github.com/Chia-Network/chia-blockchain/discussions/507#discussioncomment-526261

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
